content/departments/engineering/teams/devops/opsgenie.md

```diff
@@ -25,7 +25,7 @@ Ensure that the final escalation policy is set to "Send to the DevOps team" or "
 
 ### Creating split rotations
 
-Otherwise known as "follow-the-sun" rotations, split on-call rotations are a great way to leverage distributed, worldwide teams to make sure no one has to wake up in the middle of the night!
+Otherwise known as "follow-the-sun" rotations, split on-call rotations are a great way to leverage distributed, worldwide teams to make sure no one has to wake up in the middle of the night! If you need a "follow-the-sun" rotation with flexibility on number of teams, escalation, and rotation, [check out this approach](./opsgenie-routing.md).
```

Comment:
Note that the approach i documented has similar features - you can have arbitrary numbers of schedules, escalate to custom groups, and have custom rotations